src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -573,6 +601,22 @@ export default function EarnPoolInfoScreen({ route, navigation }: Props) {
             tokensInfo={tokensInfo}
             earnPosition={pool}
           />
+          {!!dataProps.dailyYieldRatePercentage && dataProps.dailyYieldRatePercentage > 0 && (
```

Comment:
probably fine, but should we check its >= 0.00005? so we don't end up showing 0.0000 with the round with toFixed?